release/models/nat/openconfig-nat.yang

```diff
@@ -0,0 +1,965 @@
+
+        leaf interface {
+          type leafref {
+            path "../config/interface";
+          }
+          description
+            "Reference to interface list key";
+        }
+
+        container config {
+          description
+            "Configuration parameters for the NAT interface";
+
+          uses nat-interface-config;
+        }
+
+        container state {
+          config false;
+          description
+            "Operational state data for the NAT interface";
+
+          uses nat-interface-config;
+          uses nat-interface-state;
+        }
+      }
+    }
+  }
+
+  grouping acl-set-config {
+    description
+      "Configuration parameters for NAT ACL set";
+
+    leaf acl-set {
+      type leafref {
+        path "/oc-acl:acl/oc-acl:acl-sets/oc-acl:acl-set/" +
+             "oc-acl:config/oc-acl:name";
+      }
+      description
+        "Reference to ACL set that defines traffic
+        eligible for this pool";
+    }
+  }
+
+  grouping acl-set-state {
+    description
+      "Operational state data for NAT ACL set";
+
+    container counters {
+      description
+        "Counters for ACL set operations";
+
+      leaf hit-count {
+        type yang:counter64;
+        description
+          "Number of packets matching this ACL set";
+      }
+    }
+  }
+
+  grouping nat-pool-config {
+    description
+      "Configuration parameters for NAT address pools";
+
+    leaf name {
+      type string;
+      description
+        "Unique name for the NAT pool";
+    }
+
+    leaf start-address {
+      type inet:ipv4-address;
+      description
+        "Starting IPv4 address of the pool range";
+    }
+
+    leaf end-address {
+      type inet:ipv4-address;
+      description
+        "Ending IPv4 address of the pool range";
+    }
+
+    leaf port-range-start {
+      type inet:port-number;
+      description
+        "Starting port number for NAPT translations";
+    }
+
+    leaf port-range-end {
+      type inet:port-number;
+      description
+        "Ending port number for NAPT translations";
+    }
+
+    leaf port-block-size {
+      type uint16;
+      description
+        "Number of ports allocated per mapping";
+    }
+
+    leaf acl-set {
+      type leafref {
+        path "/oc-acl:acl/oc-acl:acl-sets/oc-acl:acl-set/" +
+             "oc-acl:config/oc-acl:name";
+      }
+      description
+        "Reference to ACL set that defines traffic
+        eligible for this pool";
+    }
+
+    leaf persistent {
+      type boolean;
+      description
+        "Enable persistent NAT mapping";
+    }
+
+    leaf max-sessions {
+      type uint32;
+      description
+        "Maximum number of concurrent sessions";
+    }
+
+    leaf address-only {
+      type boolean;
+      description
+        "Enable address-only translation without port translation";
+    }
+
+    leaf timeout {
+      type uint32;
+      units "seconds";
+      description
+        "Default NAT mapping timeout";
```

Comment:
When does this default apply? 
* Does setting this modify the other timeouts? 
* Does setting this control NAT timeouts for non-TCP/UDP/ICMP traffic?